src/Grpc.NetCore.HttpClient/Grpc.NetCore.HttpClient.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Should we put a README.md under Grpc.NetCore.Http client that this is not expected to work yet?

Status and Trailers are not supported yet, so calls will always return StatusCode.OK, so (unlike Grpc.AspNetCore.Server) the managed client is not really usable yet. We should potential early adopters about this.

Reply:
Good idea